Configurations/00-base-templates.conf

```diff
@@ -43,6 +43,8 @@ my %targets=(
 	shared_ldflag	=> "",
 	shared_rcflag	=> "",
 	shared_extension	=> "",
+	
+	lib_suffix => "",
```

Comment:
You call it lib_suffix here but win_suffix in 10-main.conf.  I think you need to decide what it will be and use that consistently.